package.json

```diff
@@ -17,11 +17,12 @@
   "dependencies": {
     "addressit": "git://github.com/dianashk/addressit.git#temp",
     "extend": "^3.0.0",
-    "lodash": "^4.11.1",
+    "lodash": "^4.13.1",
+    "node-postal": "openvenues/node-postal",
+    "pelias-config": "^1.0.3",
```

Comment:
Change this to an exact version, the latest is 2.1.0: https://www.npmjs.com/package/pelias-config